Source/Core/DolphinTool/VerifyCommand.h

```diff
@@ -4,9 +4,11 @@
 #pragma once
 
 #include <iostream>
+#include <optional>
 #include <string>
 #include <vector>
 
+#include "DiscIO/Blob.h"
```

Comment:
What is this include used for?

Reply:
It is necessary to use the BlobReader methods GetBlockSize, GetCompressionMethod, and GetCompressionLevel, plus the DiscIO::BlobType enum.  If you mean why did I put it in the header instead of the cpp file, idk.